usermembership/tool/src/java/org/sakaiproject/umem/tool/ui/UserListBean.java

```diff
@@ -301,6 +300,16 @@ public String getInitValues() {
 		
 		return "";
 	}
+	
+	public TimeZone getUserTimeZone() {
+		TimeZone timeZone = userTimeService.getLocalTimeZone();
+		
+		if (timeZone == null) {
```

Comment:
I don't think this is necessary as the `UserTimeService` will return the default timezone if the user doesn't have one explicitly set.

Reply:
public TimeZone getUserTimeZone() {
		return userTimeService.getLocalTimeZone();
	}
?
delete getUserTimeZone()?

Reply:
I think it's nicer to have the method so that the JSPs are just bind against the `UserListBean` rather than having the `UserTimeService` available in the JSF scope.